e2e-tests/cypress/fixtures/appeal-statement-invalid-wrong-type.mp3

```diff
@@ -0,0 +1 @@
+some,words,for,a,dummy,test,file
```

Comment:
I'm a big enough nerd to have checked whether a text file with a `.mp3` extension is sufficient to fool the mime type checker, and it is. 

:exploding_head: